compiler/noirc_frontend/src/hir/resolution/import.rs

```diff
@@ -163,7 +163,8 @@ fn resolve_path_to_ns(
 
             let current_mod_id = ModuleId { krate: crate_id, local_id: import_directive.module_id };
             let current_mod = &def_map.modules[current_mod_id.local_id.0];
-            let first_segment = import_path.first().expect("ice: could not fetch first segment");
+            let first_segment =
+                &import_path.first().expect("ice: could not fetch first segment").ident;
```

Comment:
Should this be `import_path.first_ident()`? I don't think preserving the expect message matters.

Reply:
Could be. However this is operating on `&[PathSegment]` and not `Path`. The `Path` is available in this method, it's just that `first_ident()` returns a copy of ident, and here we just take a reference... so maybe leaving it like this is good.